cmd/juju/charmhub/data.go

```diff
@@ -43,13 +47,13 @@ type FindResponse struct {
 	StoreURL  string   `json:"store-url" yaml:"store-url"`
 }
 
-type Channel struct {
-	ReleasedAt string   `json:"released-at" yaml:"released-at"`
+type Revision struct {
 	Track      string   `json:"track" yaml:"track"`
 	Risk       string   `json:"risk" yaml:"risk"`
+	Version    string   `json:"version" yaml:"version"`
 	Revision   int      `json:"revision" yaml:"revision"`
+	ReleasedAt string   `json:"released-at" yaml:"released-at"`
```

Comment:
I reordered the fields here for what I think is a slightly nicer order in the YAML output.